common/meta/types/proto/meta.proto

```diff
@@ -76,4 +135,8 @@ service MetaService {
   // Including raft hard state, logs and state machine.
   // The exported data is a list of json strings in form of `(tree_name, sub_tree_prefix, key, value)`.
   rpc Export(Empty) returns (stream ExportedChunk);
+
+  // Add watch key stream.
+  // Whenever the watch key data updated, client will be notified accross the stream.
+  rpc Watch(stream WatchRequest) returns (stream WatchResponse);
```

Comment:
emm, etcd watch API has binary c/s stream implementation, it can save some resource if the client create/cancel/ and then create another watch request again...

Reply:
hmm... I doubt that the resource reduction is measurable in this case...
But your code your decision:DD